sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactService.java

```diff
@@ -578,6 +523,82 @@ private Predicate buildRelevantContactsFilter(Person person, Disease disease, Da
 		return Collections.emptyList();
 	}
 
+	public Predicate getRegionDistrictDiseasePredicate(
+		Region region,
+		District district,
+		Disease disease,
+		CriteriaBuilder cb,
+		Root<Contact> contact,
+		Join<Contact, Case> caze,
+		Predicate filter) {
```

Comment:
please refactor this to not use the `filter` argument

Reply:
changed